usr/src/cmd/zfs/zfs_main.c

```diff
@@ -1322,7 +1322,7 @@ zfs_do_destroy(int argc, char **argv)
 			char buf[16];
 			zfs_nicenum(cb.cb_snapused, buf, sizeof (buf));
 			if (cb.cb_parsable) {
-				(void) printf("reclaim\t%llu\n",
+				(void) printf("reclaim\t%"PRIu64"\n",
 				    cb.cb_snapused);
```

Comment:
The typical way of addressing this in ZFS has been to cast the argument to `long long`.  Personally, I find that more readable than `"PRIu64"`.

Reply:
It looks like a Hungarian notation. Both are specified in C99, do you want PRIu64 changed to long long casts?

Reply:
Yes, thank you.